src/parser/druid/feral/modules/features/checklist/Module.js

```diff
@@ -87,7 +85,6 @@ class Checklist extends BaseChecklist {
           moonfireDowngrade: this.moonfireSnapshot.downgradeSuggestionThresholds,
 
           // bloodtalons
-          predatorySwiftnessWasted: this.predatorySwiftness.suggestionThresholds,
           bloodtalonsWasted: this.bloodtalons.suggestionThresholds,
```

Comment:
If you are removing predatory swiftness suggestions from the checklist - does this mean that this is no longer an interesting metric at all, or? 

If it is no longer an interesting metric at all - the suggestions and thresholds should probably be removed from the PredatorySwiftness file too. 

If it is an interesting metric, but you are removing for some other reason, then we should address that issue instead of removing them from the checklist.

Reply:
The only use for it now would be to use instant-cast entangling roots if some mechanic requires it, or to heal with instant regrowth when you have downtime which should be almost 0 if owlweaving, which is going into moonkin to cast sunfire (and starsurge while heart of the wild is active). Without owlweaving (a ~2-5% damage loss afaik) there is quite a lot of downtime (~30% of the time spent waiting) where spending a GCD on the instant cast heal would still be of minor use. For damage there isn't any use for it anymore.

Do you only want suggestions that increase damage or also other minor benefits such as this healing? If those minor non-damage benefits aren't interesting anything involving Predatory Swiftness could be removed I assume.

Reply:
Sounds like something that could initially be removed to me. If owlweaving gets to stay around, maybe some implementation could be made to make an owlweaving build and a non-owlweaving build. We have some support to make different builds like that (holy paladin with glimmer vs no glimmer, mages with no icelance vs regular etc).